### DISCUSSION TITLE
Keep getting   'x-vercel-error': 'DEPLOYMENT_NOT_FOUND',

### DISCUSSION THREAD

**Discussion by DoglasRocha** [vercel/community#5997] [Help]
### Summary

I have an application build with SvelteKit where I perform some checks in +layout.server.js. In these checks, the program makes requests to my backend, which is hosted in Heroku.

Sometimes the application works as intended, sometimes not. The error I keep getting is:
error.response {
  status: 404,
  statusText: 'Not Found',
  headers: Object [AxiosHeaders] {
    'cache-control': 'public, max-age=0, must-revalidate',
    'content-length': '85',
    'content-type': 'application/json',
    date: 'Sun, 03 Mar 2024 03:00:47 GMT',
    server: 'Vercel',
    'strict-transport-security': 'max-age=63072000',
    'x-vercel-error': 'DEPLOYMENT_NOT_FOUND',
    'x-vercel-id': 'gru1:gru1:iad1::5s4vw-1709434847972-9cd3c9f23888',
    connection: 'close'
  },
  config: {
    transitional: {
      silentJSONParsing: true,
      forcedJSONParsing: true,
      clarifyTimeoutError: false
    },
    adapter: [ 'xhr', 'http' ],
    transformRequest: [ [Function: transformRequest] ],
    transformResponse: [ [Function: transformResponse] ],
    timeout: 0,
    xsrfCookieName: 'XSRF-TOKEN',
    xsrfHeaderName: 'X-XSRF-TOKEN',
    maxContentLength: -1,
    maxBodyLength: -1,
    env: { FormData: [Function], Blob: [class Blob] },
    validateStatus: [Function: validateStatus],
    headers: Object [AxiosHeaders] {
      Accept: 'application/json, text/plain, */*',
      'Content-Type': 'application/json',
      'User-Agent': 'axios/1.6.7',
      'Content-Length': '2',
      'Accept-Encoding': 'gzip, compress, deflate, br'
    },
    baseURL: 'https://www.a23cursinho.live/api/',
    withCredentials: true,
    method: 'post',
    url: 'admin/check',
    data: '{}'
  },

I am curious about this " 'x-vercel-error': 'DEPLOYMENT_NOT_FOUND',", because this request has nothing to do with vercel, but this error is very frequent.

I really need help with this, I searched in some foruns and have not found people with related problems.

### Steps to Reproduce

Build a SvelteKit application, where you make some requests with axios on +layout.server.js

**Comment by swarnava** [comment]
What is your application URL? It sounds like you are pointing a23cursinho.live to Vercel without any project linked to it.

**Comment by DoglasRocha** [comment]
My application url is biblioteca.a23cursinh9.live and www.a23cursinho.live points to my backend at Heroku.

**Comment by DoglasRocha** [comment]
So, I have changed the request base URL from my custom domain to the URL Heroku gives, and the issue seems to have disappeared. But I find the problem strange anyway.
